geode/src/main/java/com/yahoo/ycsb/db/GeodeClient.java

```diff
@@ -130,6 +130,7 @@ public void init() throws DBException {
     } else if (locator != null) {
       ccf.addPoolLocator(locator.getHost().getCanonicalHostName(), locator.getPort());
     }
+    ccf.setPdxReadSerialized(true);
```

Comment:
Yes this is needed all the time as the PdxInstance is required in the geode binding.
Have also changed the commit message to start with "[geode]".

Please review.

-Thanks
Prasad